### PR TITLE
Fix min/max width and make const for recentering interval

### PR DESCRIPTION
**Pull request by haydenshively**

### CODE REVIEW

File: contracts/interfaces/IAloeBlendImmutables.sol [hunk 1]
```diff
@@ -5,20 +5,24 @@ import "./ISilo.sol";
...
     /// @notice The maximum width (in ticks) of the primary Uniswap position
     function MAX_WIDTH() external view returns (uint24);
 
-    /// @notice The number of standard deviations to +/- from mean when setting primary Uniswap position
-    function B() external view returns (uint8);
-
     /// @notice The maintenance budget buffer multiplier
     /// @dev The vault will attempt to build up a maintenance budget equal to the average cost of incentivization,
     /// multiplied by K.
     function K() external view returns (uint8);
```
Comment (adamegyed) [inline contracts/interfaces/IAloeBlendImmutables.sol:21]: Is there a concise way of naming this something other than K?

File: contracts/interfaces/IAloeBlendImmutables.sol [hunk 2]
```diff
@@ -5,20 +5,24 @@ import "./ISilo.sol";
...
 
+    /// @notice The number of standard deviations to +/- from mean when setting primary Uniswap position
+    function B() external view returns (uint8);
```
Comment (adamegyed) [inline contracts/interfaces/IAloeBlendImmutables.sol:24]: Is there a reason this is simply named B? Could it have a better name?

**Review by adamegyed — COMMENTED** [review]
Left some comments. Some of the naming questions could be addressed in other PRs, but they were also modified here.

File: contracts/AloeBlend.sol [hunk 3]
```diff
@@ -489,8 +492,8 @@ contract AloeBlend is AloeBlendERC20, UniswapHelper, ReentrancyGuard, IAloeBlend
 
     /// @dev Computes position width based on sigma (volatility)
     function _computeNextPositionWidth(uint256 sigma) internal pure returns (uint24) {
-        if (sigma <= 5.024579e15) return MIN_WIDTH;
-        if (sigma >= 3.000058e17) return MAX_WIDTH;
+        if (sigma <= 1.00481445e16) return MIN_WIDTH;
+        if (sigma >= 4.41180492e17) return MAX_WIDTH;
```
[the remark this answers is not in the document]
Comment (haydenshively) [reply contracts/AloeBlend.sol:496]: Idk what I'd say other than "The volatility below which we use minimum width" and "The volatility above which we use maximum width" -- which doesn't seem too helpful. Any ideas?

File: contracts/interfaces/IAloeBlendImmutables.sol (hunk 1 above)
Comment (haydenshively) [reply contracts/interfaces/IAloeBlendImmutables.sol:21]: Idk about concise. "BUDGET_SIZE" and "BUDGET_MULTIPLIER"?? Fairly long though

File: contracts/interfaces/IAloeBlendImmutables.sol (hunk 2 above)
Comment (haydenshively) [reply contracts/interfaces/IAloeBlendImmutables.sol:24]: N for number of standard deviations? Idk

**Pull request merged**